src/plugins/links/public/content_management/save_to_library.tsx

```diff
@@ -28,11 +29,32 @@ const modalTitle = i18n.translate('links.contentManagement.saveModalTitle', {
   },
 });
 
-export const runSaveToLibrary = async (
+export const runQuickSave = async (
```

Comment:
**nit** Maybe `runSaveByValue` is a clearer name than `runQuickSave`? I'm used to "quick save" being used in the Dashboard context, which is always by reference regardless of save as vs quick save

Reply:
Oh sorry, yeah! That makes sense to me :+1: